cypress-tests/cypress/support/commands.js

```diff
@@ -2365,26 +2310,40 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add("stepUp", (stepUpBody, globalState, value) => {
-  const key = `step_up_enabled_${globalState.get("merchantId")}`;
-  stepUpBody.key = key;
-  stepUpBody.value = value;
+Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
+  let url;
+  let body;
+  const api_key = globalState.get("adminApiKey");
+  
+  switch (configType) {
+    case 'autoRetry':
+      url = `${globalState.get("baseUrl")}/configs/autoRetry`;
+      body = { ...configData, enable: value }; // enable or disable autoRetry
+      break;
+    case 'maxRetries':
+      url = `${globalState.get("baseUrl")}/configs/maxRetries`;
+      body = { ...configData, retries: value }; // set max retries
+      break;
+    case 'stepUp':
+      url = `${globalState.get("baseUrl")}/configs/stepUp`;
+      body = { ...configData, connectors: value }; // step-up configurations
+      break;
+    default:
```

Comment:
could you please add back these (blue color ones, not the red one) changes?

<img width="625" alt="image" src="https://github.com/user-attachments/assets/f3285888-2cb8-4581-bdf5-11d8aa70ee0c">

Reply:
@pixincreate  All changes done